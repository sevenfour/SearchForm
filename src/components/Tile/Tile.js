import React from 'react';
import PropTypes from 'prop-types';

import styles from './Tile.module.css';

import clsx from 'clsx';

const Tile = props => {
  const {
    children,
    className,
    ...rest
  } = props;

  return (
    <div
      className={clsx(styles.root, className)}
      {...rest}
    >
      {children}
    </div>
  );
};

Tile.propTypes = {
  children: PropTypes.node,
  className: PropTypes.string
};

export default Tile;
