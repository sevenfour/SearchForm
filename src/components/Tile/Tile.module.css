
/* stylelint-disable plugin/stylelint-no-indistinguishable-colors */

.root {
  box-shadow: var(--box-shadow);
  width: 250px;
}
/* stylelint-enable plugin/stylelint-no-indistinguishable-colors */
