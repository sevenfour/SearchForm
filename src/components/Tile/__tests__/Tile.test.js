import React from 'react';
import { shallow } from 'enzyme';

import Tile from '..';

describe('<Tile /> component tests', () => {

  it('renders', () => {
    const tile = shallow(
      <Tile />
    );

    expect(tile).toMatchSnapshot();
  });

});
