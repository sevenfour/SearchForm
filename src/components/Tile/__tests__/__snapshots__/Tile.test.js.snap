// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tile /> component tests renders 1`] = `
<div
  className="root"
/>
`;
